doc/examples/filters/plot_hysteresis.py

```diff
@@ -0,0 +1,50 @@
+"""
+=======================
+Hysteresis thresholding
```

Comment:
@soupault Although I initially thought I could do it, upon further review I think that would be quite difficult without major reworking of or addition to that example. It wouldn't help any of the "dodgier" thresholds in that application. I'm going to leave this for another occasion/contributor.